FhirToDataLake/deploy/templates/DeployAzureContainerApp.json

```diff
@@ -135,7 +135,7 @@
     },
     "image": {
         "type": "string",
-        "defaultValue": "healthplatformregistry.azurecr.io/fhirtodatalaketool:v0.5.0",
+        "defaultValue": "healthplatformregistry.azurecr.io/fhirtodatalaketool:latest",
```

Comment:
@sowu880 Still use latest when we build the v0.5.0 image on healthplatform Container Registry